tests/finish/ansible/test.sh

```diff
@@ -20,6 +20,13 @@ rlJournalStart
             if [[ $method == local ]]; then
                 rlRun "sudo rm -f /tmp/finished"
             fi
+
+            # Check that created file is synced back 
+            rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+            rlRun "tmt run -i $tmp -av finish provision -h $method -i centos:7"
```

Comment:
Tests are failing becase the `local` provision plugin does not support images:

```
:: [ 19:40:10 ] :: [  BEGIN   ] :: Running 'tmt run -i /tmp/tmp.lDOL104XYx -av finish provision -h local -i centos:7'
Usage: tmt run provision [OPTIONS]
Try 'tmt run provision --help' for help.

Error: no such option: -i
```

Let's create the `$tmp` directory already in the `Setup` phase and reuse it for all enabled provision methods using `tmt run -i $tm --scratch ...` and add the check around line 12. This will save one extra `tmt run` and will make the test twice faster.

Reply:
It's changed now.